api/src/main/java/io/opencensus/trace/Link.java

```diff
@@ -68,5 +90,12 @@ public static Link fromSpanContext(SpanContext context, Type type) {
    */
   public abstract Type getType();
 
+  /**
+   * Returns the set of attributes.
```

Comment:
why you'd put attributes on a link isn't obvious. What's the use case? Can an example end up in javadoc?